README.md

```diff
@@ -0,0 +1,139 @@
+Reference the chart directory with `chart`, and fill in appropriate
+default `values`. When looking for a value Helm will default to the
+values from the `values.yaml` in the chart directory, and the values in
+the chart temlate config entry override those.
+
+You will need to create template strings to explicitly pass through the
+values from the deployment into appropriate places in the `values`
+entry. Look at the existing entries to see how this works.
+
+## Creating a new chart
+
+Pick a name for the new chart. This needs to be unique and meaningful.
+Ideally it doesn't want to conflict with any existing ACS service or
+application. This description will use `new-app`.
+
+Create a new directory at the top of this repo named after your chart
+name. This directory contains the chart; see [the Helm
+documentation](https://helm.sh/docs/topics/charts/) for the format. The
+minumum necessary is:
+
+* `Chart.yaml`: A YAML file describing the chart. A minimal example
+  would be
+
+```yaml
+    apiVersion: v2
+    name: new-app
+    version: "0.0.1"
+    description: "ACS New App deployment"
+```
+
+* `values.yaml`: The defaults for the `values` supplied on deployment.
+  The structure here is entirely up to you; values in this file, as
+  overridden on deployment, are accessible from the `{{ .Values }}`
+  variable within the templates.
+
+* `templates`: A directory containing YAML files, with Helm templating,
+  which deploy the correct Kubernetes objects.
+
+Writing the templates requires some knowledge of the Helm template
+language. A certain amount can be picked up by looking at the existing
+charts. `helm template .` is a useful command to see what the templates
+render to and whether there are templating errors. See the Helm
+documentation for how to change the values used for rendering.
+
+The existing charts follow some conventions:
+
+* All objects are explicitly deployed into the `{{ .Release.Namespace
+  }}` namespace. This will be the namespace the edge cluster was
+  deployed to.
+
+* All objects are given names starting with the chart name and the
+  deployment UUID. This avoids conflicts. Often no other name is needed.
+
+* Objects are labelled as follows:
+
+Label|Value
+---|---
+`factory-plus.app`|Chart name
+`factory-plus.nodeUuid`|Deployment UUID
+`factory-plus.name`|Name supplied to the deployment
+
+* Where a Service is needed to provide access to a deployed container,
+  this should normally have `internalTrafficPolicy: Local`. This will
+  only make the Service available on the host the container is running
+  on. While this is not a robust security measure it helps.
+
+Once the chart is written, commit it to Git and get it pushed to your
+internal Git server. There are three ways to accomplish this:
+
+* Create a PR and get it accepted. When a new version of ACS is
+  released, upgrade to the new version. The new chart will be pulled
+  down automatically. We are unlikely to be possible for an untested
+  chart.
```

Comment:
What does `We are unlikely to be possible for an untested chart.` mean?

Reply:
It means I can't proof-read :). I'll fix it...